website/src/pages/[organism]/metadata-overview/index.ts

```diff
@@ -0,0 +1,36 @@
+import type { APIRoute } from 'astro';
+
+import { cleanOrganism } from '../../../components/Navigation/cleanOrganism.ts';
+import { getSchema } from '../../../config.ts';
+import { SUBMISSION_ID_FIELD } from '../../../settings.ts';
+
+export const GET: APIRoute = ({ params }) => {
+    const rawOrganism = params.organism!;
+    const { organism } = cleanOrganism(rawOrganism);
+    if (organism === undefined) {
+        return new Response(undefined, {
+            status: 404,
+        });
+    }
+
+    const extraFields = [SUBMISSION_ID_FIELD];
+
+    const { inputFields } = getSchema(organism.key);
+
+    const headers: Record<string, string> = {
+        'Content-Type': 'text/tsv', // eslint-disable-line @typescript-eslint/naming-convention
+    };
+
+    const filename = `${organism.displayName.replaceAll(' ', '_')}_metadata_overview.tsv`;
+    headers['Content-Disposition'] = `attachment; filename="${filename}"`;
+
+    const fieldNames = inputFields.map(
+        (field) =>
+            `${field.name}\t${field.required ?? ''}\t${field.definition ?? ''} ${field.guidance ?? ''}\t${field.example ?? ''}`,
+    );
+    const tsvTemplate = [...extraFields, ...fieldNames].join('\n');
```

Comment:
I think adding a header row would be nice, in particular for the 'required' column since it'll otherwise just be a column of boolean values and it's not clear what they mean.

Reply:
Great point! I thought this was added but it doesn't look like that - I will try to do that now!